src/Nancy/Bootstrapper/NancyBootstrapperWithRequestContainerBase.cs

```diff
@@ -189,7 +190,19 @@ protected override sealed void RegisterModules(TContainer container, IEnumerable
         /// Creates a per request child/nested container
         /// </summary>
         /// <returns>Request container instance</returns>
-        protected abstract TContainer CreateRequestContainer();
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This method is obsolete. Use the overload with NancyContext instead.")]
+        protected virtual TContainer CreateRequestContainer()
+        {
+            return this.CreateRequestContainer(null);

```

Comment:
Not sure about this, but since the framework won't call this method anymore, I don't think it matters much.